### PR TITLE
710 reset defences for sandbox and reword modal warning

### PR DESCRIPTION
**Pull request by heatherlogan-scottlogic**
## Description

Remove the part that saves sandbox defences from control & warn the user in the modal.

## Screenshots

image

## Notes

- Bullet point list with
- any notable code changes
- or explanations regarding this PR

## Concerns

- Bullet point list
- with any questions
- or concerns about this PR

## Checklist

Have you done the following?

- [x] Linked the relevant Issue
- [x] Added tests
- [x] Ensured the workflow steps are passing

### CODE REVIEW

**Review by gsproston-scottlogic — APPROVED** [review]
Very elegant!

File: backend/test/unit/controller/resetController.test.ts
Comment (pmarsh-scottlogic) [inline backend/test/unit/controller/resetController.test.ts:1]: I know not part of the ticket but can we change
```typescript
describe('handleResetAllDefences unit tests', () => {
...
```
to
```typescript
describe('handleResetProgress unit tests', () => {
...
```

**Review by pmarsh-scottlogic — CHANGES_REQUESTED** [review]
Tidy!

Just one small change from me, which isn't even to do with the ticket

**Review by pmarsh-scottlogic — APPROVED** [review]
Sweet-o

**Pull request merged**